### PR TITLE
ruby-build 20150413

### PR DESCRIPTION
**Pull request by drewish**
https://github.com/sstephenson/ruby-build/releases/tag/v20150413

### CODE REVIEW

File: Library/Formula/ruby-build.rb [hunk 1]
```diff
@@ -2,9 +2,8 @@
 
 class RubyBuild < Formula
   homepage "https://github.com/sstephenson/ruby-build"
-  url "https://github.com/sstephenson/ruby-build/archive/v20150319zf.tar.gz"
-  sha256 "5c6c34aad60df2b93e30fc2263fa400bfe20cbd4e2ceaf497431fb3568bdb738"
-  version "20150319zf"
```
Comment (drewish) [inline Library/Formula/ruby-build.rb]: I dropped the version after getting this feedback from `brew audit`:
```
ruby-build:
 * Stable: version 20150413 is redundant with version scanned from URL
```

Comment (sferik) [inline Library/Formula/ruby-build.rb]: :+1:

**Comment by mikemcquaid** [comment]
Thanks for your contribution to Homebrew! Without people like you submitting PRs we couldn't run this project. You rock!

For future reference the preferred commit message format for simple version updates is `ruby-build 20150413`.

**Comment by drewish** [comment]
Ah, thanks for the feedback on the commit title, I found the instructions for the PR title but didn't amend the commit message.

**Pull request closed without merging**